src/renderer/src/routes/Onboarding/CreateProjectScreen.tsx

```diff
@@ -0,0 +1,282 @@
+
+	const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+		const value = event.target.value
+		const localError: boolean = checkForError(
+			value.trim(),
+			PROJECT_NAME_MAX_LENGTH_GRAPHEMES,
+		)
+		setProjectName(value)
+		if (localError !== error) {
+			setError(localError)
+		}
+		setError(localError)
+	}
+
+	const graphemeCount = countGraphemes(projectName.trim())
+
+	const handleAddName = () => {
+		if (checkForError(projectName, PROJECT_NAME_MAX_LENGTH_GRAPHEMES)) {
+			setError(true)
+			return
+		}
+		setProjectNameMutation.mutate(projectName, {
+			onSuccess: () => {
+				navigate({ to: '/tab1' })
+			},
+			onError: (error) => {
+				console.error('Error setting project name:', error)
+				setErrorMessage(formatMessage(m.errorSavingProjectName))
+			},
+		})
+	}
+
+	function importConfigFile() {
+		// Placeholder for file import logic
+		setConfigFileName('myProjectConfig.comapeocat')
+	}
+
+	const topMenu = (
+		<OnboardingTopMenu
+			currentStep={3}
+			onBackPress={() =>
+				navigate({ to: '/Onboarding/CreateJoinProjectScreen' })
+			}
+		/>
+	)
+
+	return (
+		<OnboardingScreenLayout topMenu={topMenu}>
+			<StyledImage src={ProjectImage} alt="Add Project" />
+			<Text kind="title" style={{ marginTop: 12 }}>
+				{formatMessage(m.title)}
+			</Text>
+			<div
+				style={{ flexGrow: 1, overflowY: 'auto', width: '100%', marginTop: 12 }}
+			>
+				<InputWrapper>
+					<StyledTextField
+						label={formatMessage(m.enterNameLabel)}
+						required
+						placeholder={formatMessage(m.placeholder)}
+						value={projectName}
+						onChange={handleChange}
+						variant="outlined"
+						error={error}
+						sx={{
+							'& .MuiFormLabel-asterisk': {
+								color: 'red',
+							},
+						}}
+						slotProps={{
+							input: {
+								style: {
+									padding: '5px 12px',
+								},
+							},
+							inputLabel: {
+								style: { fontSize: '1.125rem' },
+							},
+							htmlInput: {
+								minLength: 1,
+							},
+						}}
+					/>
+					<CharacterCount error={error}>
+						{formatMessage(m.characterCount, {
+							count: graphemeCount,
+							maxLength: PROJECT_NAME_MAX_LENGTH_GRAPHEMES,
+						})}
+					</CharacterCount>
+				</InputWrapper>
+				{errorMessage && (
+					<Text style={{ color: RED, marginTop: 12 }}>{errorMessage}</Text>
+				)}
+				<HorizontalLine />
+				<div
+					style={{
+						width: '100%',
+						maxWidth: 400,
+						margin: '0 auto',
+						textAlign: 'center',
+					}}
+				>
+					<div
+						style={{
+							display: 'flex',
+							justifyContent: 'space-between',
+							alignItems: 'center',
+							gap: 12,
+							padding: '12px 0',
+							cursor: 'pointer',
+						}}
+						onClick={() => setAdvancedSettingOpen(!advancedSettingOpen)}
+					>
+						<Text bold style={{ fontSize: '1.125rem' }}>
+							{formatMessage(m.advancedProjectSettings)}
+						</Text>
+						{advancedSettingOpen ? <ChevronDown /> : <ChevronUp />}
+					</div>
```

Comment:
I think we should us an [accordian here](https://mui.com/material-ui/react-accordion/). Accordian would give the animation, and doesn't require us to use css to hide

Reply:
Sure. Done.